docs/hosts/lattice-protocols/rpc.md

```diff
@@ -39,9 +39,9 @@ These are the fields on the `WasmCloudEntity` structure:
 
 | Field | Type | Description |
 | :--- | --- | :--- |
-| `public_key` | String | The public key of the entity. Will begin with **M** for actors, **V** for providers |
-| `contract_id` | String | The contract ID of the entity. Left blank if this entity is an actor. |
-| `link_name` | String | Link name of the entity. Left blank if this entity is an actor. |
+| `public_key` | String | The public key of the entity. Will begin with **M** for components, **V** for providers |
+| `contract_id` | String | The contract ID of the entity. Left blank if this entity is an component. |
```

Comment:
```suggestion
| `contract_id` | String | The contract ID of the entity. Left blank if this entity is a component. |
```

Reply:
The wording around "Left blank" is something I think we should revise in a follow-up PR.